### PR TITLE
adding details about github SSH keys in contributing page

### PR DESCRIPTION
**Pull request by kevin-keraudren**
Adding mention of the github SSH keys on the contributing page, as mentioned in the issue https://github.com/scikit-image/scikit-image/issues/2071 .

### CODE REVIEW

File: .github/CONTRIBUTING.txt [hunk 1]
```diff
@@ -13,9 +13,16 @@ Here's the long and short of it:
...
       git clone git@github.com:your-username/scikit-image.git
 
-   * Add the upstream repository::
+   * Add the upstream repository:
 
-      git remote add upstream git@github.com:scikit-image/scikit-image.git
+     - If you have set up your `github SSH keys
+       <https://help.github.com/articles/adding-a-new-ssh-key-to-your-github-account/>`_::
+
+        git clone git@github.com:your-username/scikit-image.git
```
Comment (soupault) [inline .github/CONTRIBUTING.txt]: Has to be `git remote add upstream git@github.com:scikit-image/scikit-image.git`, hasn't it?

Comment (kevin-keraudren) [inline .github/CONTRIBUTING.txt]: yes, sorry I messed up the commands when adding the alternative

**Comment by soupault** [comment]
LGTM now :+1: .

**Comment by emmanuelle** [comment]
Merging, thank you @kevin-keraudren

**Comment by JDWarner** [comment]
This does address the issue. However, as stated in #2071, I strongly feel we should direct new contributors to HTTPS and remove all mention of SSH. I don't care if SSH is the "right" way, this is one place where we want the "easy" way. It's a getting started guide, not a reference.

Git itself is plenty for new contributors to wrap their heads around. No need to force them to dive into SSH keys as well. Every additional barrier we raise before their first PR is another place we'll lose potential contributors before we ever even know.

**Pull request merged**